ws-nextjs-app/cypress/support/helpers/runTestsForPage.ts

```diff
@@ -0,0 +1,18 @@
+export default ({ pageType, testSuites }) => {
+  testSuites.forEach(testData => {
+    const { path, id, service, tests } = testData;
+
+    const params = {
+      pageType,
+      path,
+      id,
+      service,
+    };
+
+    before(() => {
+      cy.visit(path);
+    });
+
+    tests.forEach(test => test(params));
```

Comment:
We could try and make this more future proof by doing the following. It would mean that if we needed to add any more params, they would automagically be passed through, without us needing to explicitly declare them...

```suggestion
    const { path, tests, ...params } = testData;

    before(() => {
      cy.visit(path);
    });

    tests.forEach(test => test({ path, ...params }));
```